ext/typeexpr/defaults_test.go

```diff
@@ -433,6 +442,98 @@ func TestDefaults_Apply(t *testing.T) {
 				}),
 			}),
 		},
+		"my_test_case": {
```

Comment:
A guess at a slightly better test case name:

```suggestion
		"nested set of objects with optional attributes, with maps as inputs": {
```